frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestHeader/ChangeRequestHeader.tsx

```diff
@@ -26,7 +26,7 @@ export const ChangeRequestHeader: FC<{ changeRequest: ChangeRequestType }> = ({
                 title={title}
                 setTitle={setTitle}
             >
-                <StyledHeader variant='h1' sx={{ mr: 1.5 }}>
+                <StyledHeader variant='h1' component='h2' sx={{ mr: 1.5 }}>
```

Comment:
In terms of page semantics, there's already an h1 on the page, so even if it's styled that way, this should be an h2.